app/src/popup.js

```diff
@@ -23,29 +24,24 @@ const echo360BaseURIs = [
   'echo360.org'
 ]
 
-function canDownload({lesson}) {
-  let downloadable = (lesson.isFuture === false && lesson.hasAvailableVideo === true && lesson.video != null && lesson.video.media && lesson.video.media.media) ? true : false;
-  return downloadable;
+function canDownload({ lesson }) {
+  return lesson.isPast && lesson.hasVideo;
```

Comment:
This might be a bug on the part of echo360 themselves, as sometimes `isPast = false`, when in reality there is a video available.

Reply:
> This might be a bug on the part of echo360 themselves, as sometimes `isPast = false`, when in reality there is a video available.

I think an option button would be helpful.

Reply:
I will look into this